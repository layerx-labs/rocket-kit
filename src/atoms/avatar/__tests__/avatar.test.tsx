import React from 'react';
import { render, screen } from '@testing-library/react';
import { Avatar } from '../../..';

describe('Avatar', () => {
  it('renders', () => {
    const { asFragment } = render(
      <Avatar alt={'avatar alt'} url={'/dummy.png'} />
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('has correct alt and src', async () => {
    const avatarAlt = 'user avatar';
    render(<Avatar alt={avatarAlt} url={'/dummy.png'} />);
    const avatar = await screen.queryByAltText(avatarAlt);
    expect(avatar).toHaveProperty('src', 'http://localhost/dummy.png');
  });
});
