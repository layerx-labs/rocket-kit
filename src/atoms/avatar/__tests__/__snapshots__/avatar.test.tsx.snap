// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders 1`] = `
<DocumentFragment>
  <img
    alt="avatar alt"
    class="sc-bdfBwQ eWkXDZ tk-avatar"
    src="/dummy.png"
  />
</DocumentFragment>
`;
